var/spack/repos/builtin/packages/encodings/package.py

```diff
@@ -21,10 +21,6 @@ class Encodings(Package):
     depends_on('util-macros', type='build')
 
     def install(self, spec, prefix):
```

Comment:
Shouldn't the `def install()` line be removed too?

Reply:
X11 fonts are installed under font-utils package's install directory.
`make install` is not installed under the packages install diretory.
If `def install` is removed, spack install is failed `Nothing was installed!`
So `def install` create fake directory.

Reply:
Unfortunately, this is another problem.  For Spack to work properly, an install directory with the same has has to always have the same files in it.  Installing stuff into the package after-the-fact breaks this paradigm, and can cause problem / unexpected consequences elsewhere.

I suppose that sin was already committed, and this PR is just updating.  But it seems to me the right way to go about this is to turn each of these font packages into a variant for `font-utils`; and then update the `font-utils` package to download them as resources (and install each downloaded font as part of its overall install).

Reply:
I rewrite font-util and remove font-*.